utils/lists.py

```diff
@@ -6,38 +6,29 @@
     "You'll be the judge", 'no... (╯°□°）╯︵ ┻━┻', 'no... baka',
     'senpai, pls no ;-;'
 ]
+i = 0
+answers = []
+questions = []
+
+
+with open ("trivia_questions_answers.txt") as fp:
+    count = 0
+    a_count = 0
+    q_count = 0
```

Comment:
All of your code looks good, however I am curious about the purpose of having an a_count and a q_count. Will the question and answer arrays not already have a count or length function that could be used to access this information? Other than that, the code looks great.

Reply:
Yeahhhh, I thought about using the length function a few days after I submitted the pull request 🤦🏼‍♂️